### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by lpatmo**
### Background/Context
Updates the PR template! Context: discussion in https://github.com/codebuddies/backend/discussions/158#discussioncomment-54738

### Before and After
Before: 
image

After:
Something closer to this:
image

### Acknowledgments (optional)
@BethanyG for the updated draft

### Other notes (optional)
N/A

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+
+Placeholder example:
+[x] Made GET /resources not protected by authorization
+[x] Changed setup for tests to make sure we're not authed for the GET requests, but are authed for POST, PATCH, and search.
+[x] Added test to make sure GET /api/v1/resources/{{guid}}/ and GET /api/v1/resources/ requests and search work without a token
+[x] Altered tests for GET GET /api/v1/resources/{{guid}}/ and GET /api/v1/resources/ requests and search to ensure that they also work with a token
+[x] Added tests to make sure PATCH and POST fail without a token
+[x] Added test to make sure DEL fails without a token
+
+## Related Tickets & Documents (Optional)
```
Comment (BethanyG) [inline .github/pull_request_template.md:47]:
```suggestion
## New Libraries/Dependancies Introduced (Fill out as needed)
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,7 +1,71 @@
...
+
+- [ ] I've updated `requirements.txt`
```
Comment (BethanyG) [inline .github/pull_request_template.md:49]:
```suggestion

```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,7 +1,71 @@
...
+
+List the new libraries here:
```
Comment (BethanyG) [inline .github/pull_request_template.md:51]:
```suggestion
If you have added libraries or other dependancies, please list them (and links to their repos) below:

- [ ] I've updated `requirements.txt`
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,7 +1,72 @@
-### Background/Context
+## What type of PR is this? (check all applicable)
 
-### Before and After
+- [ ] ♻️ Refactor
```
Comment (BethanyG) [inline .github/pull_request_template.md:3]:
```suggestion
- [ ] 🐛 Bug Fix
- [ ] ✨ Feature (_**e.g.** API change, enhancement, addition, etc._)
- [ ] ♻️ Refactor
- [ ] 📝 Documentation
- [ ] 🔖 Release
- [ ] 🚩 Other (**concerning:**          )
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -1,7 +1,72 @@
...
+- [ ] ✨ Feature (e.g. API change)
```
Comment (BethanyG) [inline .github/pull_request_template.md:4]:
```suggestion
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -1,7 +1,72 @@
...
+- [ ] 🐛 Bug Fix
```
Comment (BethanyG) [inline .github/pull_request_template.md:5]:
```suggestion
```

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -1,7 +1,72 @@
...
+- [ ] 🎨 Enhancement
```
Comment (BethanyG) [inline .github/pull_request_template.md:6]:
```suggestion
```

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -1,7 +1,72 @@
...
+- [ ] 📝 Documentation Update
```
Comment (BethanyG) [inline .github/pull_request_template.md:7]:
```suggestion
```

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -1,7 +1,72 @@
...
+- [ ] 🔖 Release
```
Comment (BethanyG) [inline .github/pull_request_template.md:8]:
```suggestion
```

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -1,7 +1,72 @@
...
+- [ ] 🚩 Other
```
Comment (BethanyG) [inline .github/pull_request_template.md:9]:
```suggestion
```

**Comment by BethanyG** [comment]
Gah.  Apologies for the individual delete suggestions.  The interface wouldn't let me do a multi-line suggestion there for some reason.  The purpose of the re-ordering is to get at what I think might be the most "common" tasks.   I also removed "enhancement".  I think "enhancements" are more or less a _feature_, the only difference being the "size" --  i.e., the addition of code that doesn't specifically address a bug.  My thinking is the fewer the categories, the easier it is to fill out.

**Comment by lpatmo** [comment]
Thanks @BethanyG for the feedback/commit suggestions!

**Pull request merged**